client/src/App.js

```diff
@@ -15,15 +16,18 @@ function App() {
           <Route exact path="/">
             <Welcome />
           </Route>
-          <Route exact path="/genres">
+          <Route exact path="/Genres">
```

Comment:
I would revert these changes. paths are usually in lower case (see other pages like reactjs.org).